### DISCUSSION TITLE
Dynamic user input form in .NET MAUI Blazor

### DISCUSSION THREAD

**Discussion by Lancelot301** [CommunityToolkit/Maui#1729] [General]
Hi 
Just wanted to know if there is a functionality or Nuget package present to create Dynamic input forms using Json or Models data annotation in .NET MAUI Blazor?

My requirement is to create a dialog form which can present certain input elements like button, Textfield, radio button, ComboBox, Datepicker etc.

Kindly help me on this.

**Comment by Jonydeere1** [comment]
I'm most interested using in MAUI Apps (android).
